libs/mini-rx-store-ng/src/lib/effects.module.ts

```diff
@@ -50,9 +50,7 @@ export class EffectsModule {
         @Optional() storeRootModule: StoreRootModule,
         @Optional() storeFeatureModule: StoreFeatureModule
     ) {
-        const effects = fromObjectsWithEffectsToEffects(
```

Comment:
![Screenshot 2022-08-13 at 22 40 06](https://user-images.githubusercontent.com/1272446/184510150-bf11d9d7-30e0-43e6-bfb1-d6dc227d0a5f.png)

Even if calling `register` multiple times, the constructor here will be called only once with the accumulated objects.

When calling register again in another module the constructor here will be called again and does not know about previously registered effects.

So I think that the splice can be safely removed.